src/timesync/timesyncd-manager.c

```diff
@@ -383,10 +383,10 @@ static int manager_adjust_clock(Manager *m, double offset, int leap_sec) {
                   "  offset       : %+.3f sec\n"
                   "  freq offset  : %+li (%i ppm)\n",
                   tmx.status, tmx.status & STA_UNSYNC ? "unsync" : "sync",
-                  tmx.time.tv_sec, tmx.time.tv_usec / NSEC_PER_MSEC,
-                  tmx.constant,
+                  (long int)tmx.time.tv_sec, tmx.time.tv_usec / NSEC_PER_MSEC,
```

Comment:
This line looks wrong to me. tv_sec is meant to be printed using `PRI_TIME` and doesn't need a cast. tv_usec is possibly `long long` and might need one to be safely printed using "%ld".